Azure/azure-cli
# [App Service] Fix #23603: `az functionapp config set`: Add new parameter to set PowerShell version

**Related command**

```
az functionapp config set
```

**Description**

PowerShell Function Apps have a configurable version configuration that can be set from the Azure Portal but isn't exposed in Azure CLI. This PR adds the option for the same.

resolves #23603

**Testing Guide**

```
az functionapp config set --resource-group {} --name {} --powershell-version 7.2
```

---

This checklist is used to make sure that common guidelines for a pull request are followed.
